README.md

```diff
@@ -1,2530 +1,30 @@
-* Documentation
+![POSIX Badge](https://img.shields.io/badge/POSIX-Compliant-green?style=flat&label=POSIX)
+![Issues Tracking Badge](https://img.shields.io/badge/issue_track-github-blue?style=flat&label=Issue%20Tracking)
+[![GitHub Build Badge](https://github.com/apache/nuttx/workflows/Build/badge.svg)](https://github.com/apache/nuttx/actions/workflows/build.yml)
+[![GitHub Documentation Build Badge](https://github.com/apache/nuttx/workflows/Build%20Documentation/badge.svg)](https://github.com/apache/nuttx/actions/workflows/doc.yml)
 
-# INTRODUCTION
 
-Apache NuttX is a real-time operating system (RTOS) with an emphasis on
-standards compliance and small footprint. Scalable from 8-bit to 64-bit
-microcontroller environments, the primary governing standards in NuttX are
-POSIX and ANSI standards. Additional standard APIs from Unix and other
-common RTOSs (such as VxWorks) are adopted for functionality not available
-under these standards, or for functionality that is not appropriate for
-deeply-embedded environments (such as fork()).
+Apache NuttX is a real-time operating system (RTOS) with an emphasis on standards compliance and small footprint. Scalable from 8-bit to 64-bit microcontroller environments, the primary governing standards in NuttX are POSIX and ANSI standards. Additional standard APIs from Unix and other common RTOSs (such as VxWorks) are adopted for functionality not available under these standards, or for functionality that is not appropriate for deeply-embedded environments (such as fork()).
 
-Extensive documentation can be found on the project wiki:
-  <https://cwiki.apache.org/NUTTX/NuttX>
+For brevity, many parts of the documentation will refer to Apache NuttX as simply NuttX.
 
-For brevity, many parts of the documentation will refer to Apache NuttX as
-simply NuttX.
+## Getting Started
+First time on NuttX? Read the [Getting Started](https://nuttx.apache.org/docs/latest/quickstart/index.html) guide!
+If you don't have a board available, NuttX has its own simulator that you can run on terminal.
 
-# COMMUNITY
+## Documentation
+You can find the current NuttX documentation on: https://nuttx.apache.org/docs/latest/
 
-Every volunteer project obtains its strength from the people involved in
-it.  We invite you to participate as much or as little as you choose.
+Alternatively, you can build the documentation yourself by following the Documentation Build [Instructions](https://nuttx.apache.org/docs/latest/contributing/documentation.html).
 
-We encourage you to:
+The old NuttX documentation is still available in the [Apache wiki](https://cwiki.apache.org/NUTTX/NuttX).
 
-  - Use our project and provide feedback.
-  - Provide us with use-cases.
-  - Report bugs and submit patches.
-  - Contribute code or documentation.
+## Supported Boards
+NuttX supports a wide variety of platforms. See the full list on the [Supported Platforms](https://nuttx.apache.org/docs/latest/introduction/supported_platforms.html) page.
```

Comment:
I think the only reliable source of supported platforms/boards is `boards/` directory. To be honest, this is the first time I see this subpage and I thought that the supported platforms where on https://nuttx.apache.org/docs/latest/platforms/index.html
So it turns out that we have supported platforms in 2 different places and neither of them is complete. What a mess :)

Reply:
Good catch. I think the documentation page could benefit from an update. Even upgrading the looks to something more modern could be done

Reply:
https://nuttx.apache.org/docs/latest/introduction/supported_platforms.html has been removed in https://github.com/apache/nuttx/commit/88b7ce80a00d8c75beaa530ce1ca3611b02cd8b9 so please link to https://nuttx.apache.org/docs/latest/platforms/index.html